README.md

```diff
@@ -424,6 +424,24 @@ A number of options are available to allow you to customise the SDK:
   | `borderRadiusButton`                    | `string`   | Change border radius value of Primary, Secondary and Document Type Option buttons                                                           |
   | `buttonGroupStacked` (default: `false`) | `boolean`  | Display Primary, Secondary button group in Document and Face capture confirmation screens are in separate rows instead of inline by default |
 
+  | Links                       | Description                               |
+  | --------------------------- | ----------------------------------------- |
+  | `colorContentLinkTextHover` | Change Link text color                    |
+  | `colorBorderLinkUnderline`  | Change Link underline color               |
+  | `colorBackgroundLinkHover`  | Change Link background color on hover     |
+  | `colorBackgroundLinkActive` | Change Link background color on click/tap |
+
+  | Warning, Error Popups              | Description                                                |
+  | ---------------------------------- | ---------------------------------------------------------- |
+  | `colorContentAlertInfo`            | Change warning popup fallback Link underline color         |
+  | `colorBackgroundAlertInfo`         | Change warning popup fallback Link background on hover     |
+  | `colorContentAlertInfoLinkHover`   | Change warning popup fallback Link background on click/tap |
+  | `colorContentAlertInfoLinkActive`  | Change warning popup fallback Link background on click/tap |
+  | `colorContentAlertError`           | Change error popup fallback Link underline color           |
+  | `colorBackgroundAlertError`        | Change error popup fallback Link background on hover       |
+  | `colorContentAlertErrorLinkHover`  | Change error popup fallback Link background on click/tap   |
+  | `colorContentAlertErrorLinkActive` | Change error popup fallback Link background on click/tap   |
```

Comment:
I think there may been some copy/paste errors here? The two options with `Hover` in the name mention `click/tap` in the description and there are two options with `hover` in the description but not in the name.

Also `Content` in the name of some but `background` in the description.

Reply:
Oops. Thanks for catching this! Thought I cleaned it up, but guess not. 🙈